configs/common.config

```diff
@@ -39,7 +39,7 @@ CT_LIBC_NEWLIB_RETARGETABLE_LOCKING=y
 CT_LIBC_NEWLIB_EXTRA_SECTIONS=y
 
 # Newlib-nano
-CT_COMP_LIBS_NEWLIB_NANO=y
+CT_COMP_LIBS_NEWLIB_NANO=n
```

Comment:
I have merged https://github.com/zephyrproject-rtos/sdk-ng/pull/828, which increases the toolchain build job timeout to 12 hours; so, this can be removed now.

Reply:
Ooo!

Reply:
8 hours for `arm-zephyr-eabi` ... at least, it is not over 10 hours